web/frontend/src/App.tsx

```diff
@@ -1,46 +1,15 @@
-import { Suspense, lazy, useState } from 'react';
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import CognitoProvider from '@/features/cognito/cognito-hosted';
-import Header from '@/components/Header';
-import Footer from '@/components/Footer';
-import FeedbackButton from '@/components/FeedbackButton';
-import './index.css';
-import './fonts.css';
+import { TRIApp } from '@toyota-research-institute/rse-react-library';
+import '@toyota-research-institute/rse-react-library/style.css';
+import config from './tri.app.config';
+import Prelogin from '@/pages/pre-login/pre-login';
 
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
```

Comment:
`TRIApp` actually handles instantiating a `queryClient`, so I think you can skip this import and the associated logic in this file.

Reply:
I had removed this in the react library cause I wasn't sure if all projects will use `use-query`. We can add it back later if it seems all projects use it anyways.